src/screenipy.py

```diff
@@ -79,14 +85,50 @@ def initExecution():
         Utility.tools.clearScreen()
         return initExecution()
 
+
+def isTradingTime():
+    curr = datetime.now(pytz.timezone('Asia/Kolkata'))
+    openTime = curr.replace(hour=9, minute=15)
+    closeTime = curr.replace(hour=15, minute=30)
+    isTradingTime = ((openTime <= curr <= closeTime) and (
+        0 <= curr.weekday() <= 4))
+
+
+def saveStockData():
+    global stockDict
+    today_date = date.today().strftime("%d%m%y")
+    cache_file = "stock_data_" + str(today_date) + ".pkl"
+    if not os.path.exists(cache_file):
```

Comment:
# Need BUGFIX
If We abort Script using `Ctrl+C` Data pickled for 1st time. But For next screen, even though script fetches data from both dictionary and then look for online data for rest of the stock codes, this newly fetched results don't get saved!
## Example:
**Disable `stockShuffling` from Configuration to reproduce this bug clearly**
* Init with No cached data. 
* Pressed `Ctrl+C` after screening 100 stocks. This saves 100 stocks to cached pickle file.
* Run screener again with any option. This loads first 100 stocks from cache and then goes online to look for rest of the stocks
* Press `Ctr+C` again say after 200 stocks.
* But at this time, the file with today's datestamp **already exists!** So it doesn't save the newly fetched stocks. Ideally it should replace the existing file.

Reply:
@swarpatel23 Please test ddf343a to verify this bugfix